docs/articles/developer-info/Issue-Tracking.md

```diff
@@ -72,9 +72,9 @@ The goal with these is to keep team members from having to do tedious rescanning
 
 Labels starting with **closed:** indicate the status of the bug at closing and should only appear on closed bugs. Please remember to apply one of these when closing a bug as it makes it easier to review the list of closed bugs without opening each one to see what the disposition was.
 
-* **closed:done** The work called for is done, i.e. the bug is fixed or the feature/enhancement is implemented.
+* **`closed:done** The work called for is done, i.e. the bug is fixed or the feature/enhancement is implemented.
```

Comment:
Is the ` on purpose? Same comment for line 77

Reply:
Nope! Definitely a mistake. Saw your comment when I came here to review for those sorts of issues. 👍 thanks! Will fix.